### PR TITLE
[7084] XB styles are affected by Guest app styles

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/7084

### CODE REVIEW

File: ui/guest/src/react/GuestGlobalStyles.tsx [hunk 1]
```diff
@@ -62,184 +63,187 @@ export function GuestGlobalStyles(props: GuestGlobalStylesProps) {
...
-            }
-          },
-          styles ?? {}
-        )}
-      />
+            },
+            styles ?? {}
+          )}
+        />
+        <Global styles={staticGlobalStyles} />
```
Comment (rart) [inline ui/guest/src/react/GuestGlobalStyles.tsx]: These styles have more than just the typography. Let's split typography vs other things on the studio Side, so you can import `typograhyStyles` by itself. Also, on the Studio side, where ever those things are use, you'd now need to create 2 `<Global />` blocks to add both.

File: ui/guest/src/react/UltraStyledTooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+import { Tooltip, TooltipProps } from '@mui/material';
+
+export const UltraStyledTooltip = ({ title, children, ...props }: TooltipProps) => {
+  return (
+    <Tooltip slotProps={{ tooltip: { sx: { fontSize: 12 } } }} {...props} title={title}>
+      {children}
+    </Tooltip>
+  );
+};
```
Comment (rart) [inline ui/guest/src/react/UltraStyledTooltip.tsx]: Why not done through the styled API?

File: ui/guest/src/react/UltraStyledTypography.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+// Since this typography runs guest-side, there may be all sorts of styles that affect Typography styles.
+// The idea of this component is to be as specific as possible to avoid guest site styles to break ours.
+
+import { styled } from '@mui/material/styles';
+import { Typography } from '@mui/material';
+
+export const UltraStyledTypography = styled(Typography)(({ theme }) => ({
+  color: theme.palette.text.primary,
+  letterSpacing: 'unset',
+  fontSize: 16
```
Comment (rart) [inline ui/guest/src/react/UltraStyledTypography.tsx]: Isn't the font size by default 14 px? Actually, we can access that from the theme object? (i.e. `theme.typography.fontSize`)

**Review by rart — CHANGES_REQUESTED** [review]

File: ui/guest/src/react/UltraStyledTooltip.tsx (hunk 2 above)
Comment (jvega190) [reply ui/guest/src/react/UltraStyledTooltip.tsx]: For some reason the styles for the popper are not applying through sx styles. I tried on an example provided from mui.com and doesn't work. https://codesandbox.io/s/jovial-wescoff-23jhhj?file=/src/Demo.tsx
Tooltip classes doc: https://mui.com/material-ui/api/tooltip/#classes.

Comment (jvega190) [reply ui/guest/src/react/UltraStyledTooltip.tsx]: Using slotProps to set the styles for the tooltip works

Comment (rart) [inline ui/guest/src/react/UltraStyledTooltip.tsx]:
```suggestion
export const UltraStyledTooltip = (props: TooltipProps) => {
  return (
    <Tooltip {...props} slotProps={{ ...props?.slotProps, tooltip: { ...props.slotProps?.tooltip, sx: { ...props.slotProps?.tooltip?.sx, fontSize: 12 } } }} />
  );
};
```

**Review by rart — CHANGES_REQUESTED** [review]

**Review by rart — APPROVED** [review]

**Pull request merged**